docs/release_notes.rst

```diff
@@ -12,6 +12,7 @@ Post v1.4 :small:`May 13, 2019`
 
 New functionality:
 
+- New module :ref:`sc.get<module-get>` adds helper functions for extracting data in convenient formats :pr:`619` :smaller:`thanks to I Virshup`
```

Comment:
why not `` :mod:`scanpy.get` ``?

Reply:
I think it's more useful if it links to the docs, which ``:mod:`scanpy.get` `` didn't seem to do.

Reply:
It could definitely get changed to ``:ref:`scanpy.get<module-get>` ``, though I'm not sure if how the styling can be fixed to look like a module.

Reply:
It can’t. We should fix that `` :mod:`scanpy.get` `` doesn’t link to the proper location.

I guess that could be done via [`.. module:: scanpy.get`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-py:module)

Reply:
Fixed it! 0485cb8c5d28e7316622a2d8c26e538014506e68

I just put the module link targets at the correct position and immediately reset the “current module” to scanpy each time, so that relative links still work!